src/addons/addonmessage.h

```diff
@@ -19,7 +23,7 @@ class AddonMessage final : public Addon {
   ADDON_PROPERTY(subtitle, m_subtitle, retranslationCompleted)
 
   Q_PROPERTY(Composer* composer READ composer CONSTANT)
-  Q_PROPERTY(bool isRead MEMBER m_isRead NOTIFY isReadChanged)
+  Q_PROPERTY(bool isRead READ isRead)
   Q_PROPERTY(qint64 date MEMBER m_date)
```

Comment:
Hm, I don't think I can. 

> The presence of the CONSTANT attribute indicates that the property value is constant. For a given object instance, the READ method of a constant property must return the same value every time it is called. This constant value may be different for different instances of the object. A constant property cannot have a WRITE method or a NOTIFY signal.

This value will require a `NOTIFY` and is also not a constant value, it can change.

Reply:
no `m_date` cannot change. It's the date of the message and it's read from the manifest. It should be set as CONSTANT.

Reply:
Oh! You were referring to `m_date` hehe I thought you were referring to `isRead`